components/brave_ads/browser/ads_service_impl_unittest.cc

```diff
@@ -238,6 +239,9 @@ class MockRewardsService : public RewardsService {
                void(brave_rewards::GetWalletPassphraseCallback));
 
   MOCK_METHOD1(SetAdsEnabled, void(const bool is_enabled));
+  MOCK_METHOD0(GetExternalWalletProviders, base::Value());
```

Comment:
Does the signature need to be `std::vector<std::string>()` now?

Reply:
resolved